ONBOARDING.md

```diff
@@ -3,67 +3,41 @@
 This document is an outline of the things we tell new Build WG members at their
 onboarding session.
 
-### Before the onboarding session
+## Onboarding checklists
 
-* Email the new member and obtain their public GPG key
-* Add them to the `test` group in `nodejs-private/secrets`
-* Add them to the `nodejs/build` and `nodejs-private/build` GitHub teams
-* Ask them to start looking over `services.md` and `jenkins-guide.md`,
-  and to try and have any questions ready
-* Ask the onboardee to install:
-  * Ruby and the `dotgpg` gem
-  * Python 2.7 and Ansible
+### Onboarding to build-test issue checklist
 
-### General information
-
-* Responsibilities: The Build WG manages everything in
-  [`doc/services.md`](doc/services.md)
-* Members/"who does what"/interest areas
-* Technologies and software in use
+- [ ] Email the new member and obtain their public GPG key
+- [ ] Add them to the `test` group in `nodejs-private/secrets`
+- [ ] Add them to the `nodejs/build` and `nodejs-private/build` GitHub teams
+- [ ] Schedule a meeting with the member to:
+    - [ ] Walk them through the infrastructure and what other members do 
+    - [ ] Explain how to decrypt nodejs/secrets 
+    - [ ] Practice running the `jenkins/worker/create.yml` playbook
+  on one of the machines in the test CI cluster
+    - [ ] Answer any questions they may have
+- [ ] Before the meeting the Onboardee is to:
+  - [ ] Install Ruby and the `dotgpg` gem
+  - [ ] Install Python and Ansible
+  - [ ] Read `services.md`. `jenkins-guide.md`,
+    `GOVERNANCE.md` and to try and have any questions ready
+- [ ] PR changes to [README.md](./README.md#build-wg-members) to add the member to build-test
 
-### CI/Jenkins
 
-* Public CI (ci.nodejs.org)
-  * `nodejs/jenkins-admins` are administrators
-  * Tests Node.js itself, and several other Foundation projects
-* Release CI (ci-release.nodejs.org)
-  * `nodejs/jenkins-release-admins` are administrators
-  * What creates and publishes all releases of Node.js
+All members that are onboarded to the build-release and build-infra teams
```

Comment:
I don't know if its useful to have these last couple. For one thing, the emailing is unnecessary, the key are already in nodejs/secrets/build/test (assuming that no one is added to the more critical release or infra if they aren't already in build, which is always true AFAIK). For another, there are lots of little bits of extra permissions that can be given, maybe best just list them all as a single checklist?